src/java.base/unix/classes/java/lang/ProcessEnvironment.java

```diff
@@ -103,22 +112,22 @@
         } else {
             tracePrunedEnvVarsValue = 0;
         }
-        // We cache the C environment.  This means that subsequent calls
-        // to putenv/setenv from C will not be visible from Java code.
-        byte[][] environ = environ();
         origEnvironment = new HashMap<>(environ.length/2 + 3);
+        Variable tmpKeyVar;
```

Comment:
Please remove this line and move the declaration to line 125.

Reply:
moved